lib/WindowsCompat.js

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
Problably in case more compat functions get added. But exporting named functions cuts it too.